pkg/keys/ecdsa.go

```diff
@@ -40,15 +40,14 @@ type EcdsaVerifier struct {
 	key       *data.PublicKey
 }
 
-func (p *EcdsaVerifier) Public() string {
+func (p *EcdsaVerifier) Public() (string, error) {
 	// This is already verified to succeed when unmarshalling a public key.
 	r, err := x509.MarshalPKIXPublicKey(p.ecdsaKey)
 	if err != nil {
-		// TODO: Gracefully handle these errors.
```

Comment:
How about:

```go
if err != nil {
    panic(err) // This is already verified to succeed when unmarshalling a public key.
}
```

Reply:
I'm formatting the error with `tuf: error unmarshalling key` as recommended by @asraa  in the [issue](https://github.com/theupdateframework/go-tuf/issues/363)

Reply:
also would it be helpful to return the faulty key alongside the error? Like, `tuf: error unmarshalling key: "hhdheu29394..." error`

Reply:
perfect! i don't think there's any need to stick the key in there, what you have is great